src/pages/index.js

```diff
@@ -1,45 +1,38 @@
-import "./index.css";
-import { enableValidation, validationConfig } from "../scripts/validation.js";
+import "../pages/index.css";
+import { enableValidation, resetValidation, validationConfig } from "../scripts/validation.js";
+import Api from "../utils/Api.js";
 
-const initialCards = [
-  {
-    name: "Val Thorens",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/1-photo-by-moritz-feldmann-from-pexels.jpg",
-  },
-  {
-    name: "Restaurant terrace",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/2-photo-by-ceiline-from-pexels.jpg",
-  },
-  {
-    name: "An outdoor cafe",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/3-photo-by-tubanur-dogan-from-pexels.jpg",
-  },
-  {
-    name: "A very long bridge, over the forest and through the trees",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/4-photo-by-maurice-laschet-from-pexels.jpg",
-  },
-  {
-    name: "Tunnel with morning light",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/5-photo-by-van-anh-nguyen-from-pexels.jpg",
-  },
-  {
-    name: "Mountain house",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/6-photo-by-moritz-feldmann-from-pexels.jpg",
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "30d363a4-0523-477a-9b2a-fd32a4407d5b",
+    "Content-Type": "application/json"
   },
-];
+});
+
+api.getAppInfo().then(res => {
+  res[0].forEach((item) => {
+      const cardElement = getCardElement(item);
+      cardsList.append(cardElement);
+  });
+
+  profileName.textContent = res[1].name;
+  profileDescription.textContent = res[1].about;
+  document.querySelector(".profile__avatar").src = res[1].avatar;
```

Comment:
`document.querySelector(".profile__avatar")`


You've already found `avatarImage` in line 54. Please, use it here


```js

  avatarImage.src = res[1].avatar;

```